generated-platform-project/quarkus-dependencies-to-build/pom.xml

```diff
@@ -68,7 +68,7 @@
                   </goals>
                   <configuration>
                     <bom>io.quarkus.platform:quarkus-bom:${project.version}</bom>
-                    <outputFile>..\..\target\dependencies-to-build/quarkus-bom-deps-to-build.txt</outputFile>
+                    <outputFile>../../target/dependencies-to-build/quarkus-bom-deps-to-build.txt</outputFile>
```

Comment:
@aloubyansky I added a separate commit so that you can see what needs fixing for Windows in the generator.